misc.inc.php

```diff
@@ -923,16 +923,18 @@ function BuildCabinet($cabid,$face="front"){
 	$dev->ParentDevice=0;
 	$bounds=array(
 		'max'=>array('position'=>0,'height'=>0),
-		'min'=>array('position'=>0,'height'=>0),
+		'min'=>array('position'=>$cab->StartUNum,'height'=>0),
 	);
 
 	// Read in all the devices and make sure they fit the cabinet.  If not expand it
 	foreach($dev->Search() as $device){
-		if($device->Position==0){
```

Comment:
No, this need for visible errors and for devices draggable out of borders rows.

Example:
![scr6](https://cloud.githubusercontent.com/assets/17050896/13110560/e5197de4-d591-11e5-8c7e-fc4e683664aa.png)